.gitignore

```diff
@@ -142,3 +142,5 @@ ingestion_server/ingestion_server/lock
 .vscode
 
 openapi.yaml
+Pipfile.lock
+Pipfile
```

Comment:
Similar to #1008, these two files cannot be ignored (see [this comment](https://github.com/WordPress/openverse-api/pull/1008#discussion_r1022917053)).